src/Shop/Cart/Cart.php

```diff
@@ -14,7 +14,7 @@ class Cart
 
     /**
      * @param Customer\Id $customerId
-     * @param Items[] $item
+     * @param Item[] $item
```

Comment:
Oops, my fault again.

**Tip for those fixes:** Make a fresh branch from `master`.  Fix those things unrelated to your working branch. Then merge it back into `master` and rebase your working branch against the now fixed `master` branch. If you don't want to wait for tests or review you can skip the merge into `master` branch and _optimistically_ rebase against you working branch.